scipy/linalg/_sketches.py

```diff
@@ -35,42 +37,38 @@ def cwt_matrix(n_rows, n_columns, seed=None):
 
     Returns
     -------
-    S : (n_rows, n_columns) array_like
+    S : (n_rows, n_columns) csc_matrix
+        The returned matrix has ``n_columns`` nonzero entries.
 
     Notes
     -----
     Given a matrix A, with probability at least 9/10,
-    .. math:: ||SA|| == (1 \pm \epsilon)||A||
-    Where epsilon is related to the size of S
+    .. math:: \|SA\| = (1 \pm \epsilon)\|A\|
+    Where the error epsilon is related to the size of S
     """
-    S = np.zeros((n_rows, n_columns))
-    nz_positions = np.random.randint(0, n_rows, n_columns)
     rng = check_random_state(seed)
-    values = rng.choice([1, -1], n_columns)
-    for i in range(n_columns):
-        S[nz_positions[i]][i] = values[i]
-
-    return S
+    rows = rng.randint(0,n_rows,n_columns)
```

Comment:
Style nitpick: prefer spaces after commas.

Reply:
Fixed in most recent commit. Thanks!